packages/backend/src/api/endpoints/SamlConnectionApi.ts

```diff
@@ -0,0 +1,90 @@
+import type { SamlIdpSlug } from '@clerk/types';
+
+import { joinPaths } from '../../util/path';
+import type { SamlConnection } from '../resources';
+import { AbstractAPI } from './AbstractApi';
+
+const basePath = '/saml_connections';
+
+type SamlConnectionListParams = {
+  limit?: number;
+  offset?: number;
+};
+type CreateSamlConnectionParams = {
+  name: string;
+  provider: SamlIdpSlug;
+  domain: string;
+  idp_entity_id?: string;
+  idp_sso_url?: string;
+  idp_certificate?: string;
+  idp_metadata_url?: string;
+  attribute_mapping?: {
+    email_address?: string;
+    first_name?: string;
+    last_name?: string;
+    user_id?: string;
+  };
+};
+
+type UpdateSamlConnectionParams = {
+  name?: string;
+  provider?: SamlIdpSlug;
+  domain?: string;
+  idp_entity_id?: string;
+  idp_sso_url?: string;
+  idp_certificate?: string;
+  idp_metadata_url?: string;
+  attribute_mapping?: {
```

Comment:
Same here, these need to be camelCased. See an example here: https://github.com/clerk/javascript/blob/966b312050b572fbbbc07a6f0581cbec21847375/packages/backend/src/api/endpoints/UserApi.ts#L76